noxfile.py

```diff
@@ -35,3 +35,51 @@ def mypy(session: nox.Session):
 def test(session: nox.Session):
     session.install("pytest", ".")
     session.run("pytest", "setuptools_rust", *session.posargs)
+
+
+@nox.session(name="test-examples-emscripten")
+def test_examples_emscripten(session: nox.Session):
+    session.install(".")
+    emscripten_dir = Path("./emscripten").resolve()
+
+    session.run(
+        "rustup",
+        "component",
+        "add",
+        "rust-src",
+        "--toolchain",
+        "nightly",
+        external=True,
+    )
+    examples_dir = Path("examples")
+    test_crates = [
+        examples_dir / "html-py-ever",
+        examples_dir / "namespace_package",
+    ]
+    for example in test_crates:
+        env = os.environ.copy()
+        env.update(
+            RUSTUP_TOOLCHAIN="nightly",
+            _SETUPTOOLSRUST_BUILD_STD="1",
+            PYTHONPATH=str(emscripten_dir),
+            _PYTHON_SYSCONFIGDATA_NAME="_sysconfigdata__emscripten_wasm32-emscripten",
+            _PYTHON_HOST_PLATFORM="emscripten_3_1_14_wasm32",
+            CARGO_BUILD_TARGET="wasm32-unknown-emscripten",
+            CARGO_UNSTABLE_BUILD_STD="true",
```

Comment:
Looking at the CI failure and [docs for `build-std`](https://doc.rust-lang.org/cargo/reference/unstable.html#build-std) I think this actually might want to be `CARGO_UNSTABLE_BUILD_STD="core,alloc,std,proc_macro,test"`. Or presumably we can drop this soon-ish when your upstream Rust PRs merge.